IPython/core/tests/test_alias.py

```diff
@@ -38,4 +38,25 @@ def test_alias_args_error():
     with capture_output() as cap:
         _ip.run_cell('parts 1')
 
-    nt.assert_equal(cap.stderr.split(':')[0], 'UsageError')
\ No newline at end of file
+    nt.assert_equal(cap.stderr.split(':')[0], 'UsageError')
+    
+def test_alias_args_commented():
+    """Check that alias correctly ignores 'commented out' args"""
+    _ip.magic('alias commetarg echo this is %%s a "commented out" arg')
+    
+    with capture_output() as cap:
+        _ip.run_cell('commetarg')
+    
+    nt.assert_equal(cap.stdout, 'this is %s a "commented out" arg')
```

Comment:
This is failing, because the quotes disappear at some point. It's probably best just to leave the quotes out of the test case altogether.

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/ipython/ipython/IPython/core/tests/test_alias.py", line 50, in test_alias_args_commented
  nt.assert_equal(cap.stdout, 'this is %s a "commented out" arg')
nose.proxy.AssertionError: 'this is %s a commented out arg' != 'this is %s a "commented out" arg'
- this is %s a commented out arg
+ this is %s a "commented out" arg
```